05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,36 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.big {
+  width:500px;
+  float:left;
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+img {
+  height:100%;
+  width:100%;
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+#ipsum {
+  width:500px;
+  float:right;
+  background-color: rgb(240,240,240);
+  height:398px;
+  font-color:#252b3b;
+  text-align: center;
 }
 
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+p {
+  font-color: #555;
+  padding-bottom: 20px;
 }
 
-.left h1 {
-    text-align: center;
+h2 {
+  padding-top: 50px;
+  font-size: 1.9em;
 }
 
-.left p {
-    text-align: center;
+a {
+  font-weight: Bold;
```

Comment:
font-weight uses numerical values. bold starts at 700. the increments are in 100s.